README.md

```diff
@@ -5,6 +5,8 @@
 [![Release](https://img.shields.io/github/v/release/mattermost/mattermost-plugin-zoom)](https://github.com/mattermost/mattermost-plugin-zoom/releases/latest)
 [![HW](https://img.shields.io/github/issues/mattermost/mattermost-plugin-zoom/Up%20For%20Grabs?color=dark%20green&label=Help%20Wanted)](https://github.com/mattermost/mattermost-plugin-zoom/issues?q=is%3Aissue+is%3Aopen+sort%3Aupdated-desc+label%3A%22Up+For+Grabs%22+label%3A%22Help+Wanted%22)
 
+![Mattermost Plugin dashboard](https://raw.githubusercontent.com/mattermost/mattermost-plugin-zoom/e1d5480691d4c184d028ea4028eae7421810729b/public/setup_flow_images/oauth_scope_summary.png)
```

Comment:
This image needs to be stored in the `assets` folder in this repo. The reference will then look something like this: `![](.gitbook/assets/example.png)`. Let me know if you have any trouble with that! :)